### PR TITLE
Add support for radio and checkbox inputs to ajax-form

### LINKED ISSUE #570: ajaxform component doesn't work w/ radio inputs
Because of the way radio buttons work, and the way our ajaxform component needs to manually populate a `FormData` instance (see #520), our ajaxform doesn't set the correct value for a radio input.  Instead, it will submit form data such that it always looks like the very last radio button in a group has been selected (I think).

This isn't a problem _for now_ because we don't have radios in our one ajaxform.  But we might want to at least detect the radio situation and throw an exception if it looks like there's a radio in the ajaxform or something (though obviously, even better would be to just support radios).

### PR DESCRIPTION
**Pull request by jseppi**
This PR adds `input type="radio"` and `input type="checkbox"` support to our `ajax-form` web component.

I've also created a `UswdsCheckbox` widget for WDS-style checkboxes, in the same vein as our `UswdsRadio` widget. Styleguide examples have been updated to include some checkbox inputs.

Closes #570 

Needed for #985 

TO DO:
- [x] verify that the formData for checkboxes can be read by the Django backing form
- [x] figure out what to do about `<select>` and `<textarea>` form elements

### CODE REVIEW

File: frontend/source/js/data-capture/ajaxform.js [hunk 1]
```diff
@@ -94,9 +94,6 @@ class AjaxForm extends window.HTMLFormElement {
 
       if (el.isUpgraded) {
         formData.append(el.name, el.upgradedValue);
-      } else if (el.type === 'radio' || el.type === 'checked') {
```
Comment (jseppi) [inline frontend/source/js/data-capture/ajaxform.js]: I may have missed this before, but should that `el.type === 'checked'` clause instead have read `el.type === 'checkbox'`?

**Comment by jseppi** [comment]
So, it looks like `ajax-form` will not work with `<select>` or `<textarea>` elements. Should we file an issue and/or put some error-throwing code in `ajaxform.js` to help out future devs?

**Comment by jseppi** [comment]
Ready for review, @toolness!

File: frontend/source/js/data-capture/ajaxform.js (hunk 1 above)
Comment (toolness) [inline frontend/source/js/data-capture/ajaxform.js]: Oh heh, [yup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox)! My bad.

**Comment by toolness** [comment]
> So, it looks like `ajax-form` will not work with `<select>` or `<textarea>` elements. Should we file an issue and/or put some error-throwing code in `ajaxform.js` to help out future devs?

Yup that sounds good to me!

**Comment by toolness** [comment]
This looks rad! 👍

**Pull request merged**